### PR TITLE
Updated label_quality_utils.py so that Sphinx automatically inserts types

### PR DESCRIPTION
**Pull request by ethanotran**
label_quality_utils.html before PR: 

Screenshot (685)

label_quality_utils.html after PR: 

Screenshot (686)

### CODE REVIEW

**Comment by jwmueller** [comment]
@ethanotran You'll need to fill out the CLA linked above before the workflow for this PR can properly run. Thanks!

**Comment by jwmueller** [comment]
There also should not be 71 files changed. You have added far too many files to git tracking that are not related to the cleanlab codebase.

File: cleanlab/internal/label_quality_utils.py [hunk 1]
```diff
@@ -96,17 +96,17 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
...
 
-    min_allowed_prob : float, default=1e-6
+    min_allowed_prob
       Minimum allowed probability value. Entries of `pred_probs` below this value will be clipped to this value.
       Ensures entropy remains well-behaved even when `pred_probs` contains zeros.
 
     Returns
     -------
-    entropy : np.ndarray (float)
+    entropy : np.ndarray
```
Comment (jwmueller) [inline cleanlab/internal/label_quality_utils.py:109]: You can also remove the type annotation for this argument in its docstring here. Note that it is automatically added under "Return Type" already.

**Comment by ethanotran** [comment]
ok I'll take care of the additional files... am I on the right track when it comes to updating files like label_quality_utils.py?

**Comment by jwmueller** [comment]
> ok I'll take care of the additional files... am I on the right track when it comes to updating files like label_quality_utils.py?

Yep that looks on track to me!  As you go through the codebase in future PRs, you'll need to be careful when it comes to custom / less-standard types, so I wouldn't touch those for now.  Before you start editing other files though, let's first just complete the PR for label_quality_utils.py to go through the complete process of adding a little bit of new code.

File: cleanlab/internal/label_quality_utils.py [hunk 2]
```diff
@@ -96,17 +96,17 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
 
     Parameters
     ----------
-    pred_probs : np.ndarray (shape (N, K))
+    pred_probs
       Each row of this matrix corresponds to an example x and contains the model-predicted
       probabilities that x belongs to each possible class: P(label=k|x)
 
-    min_allowed_prob : float, default=1e-6
+    min_allowed_prob
```
Comment (jwmueller) [inline cleanlab/internal/label_quality_utils.py:103]: Note this still doesn't exactly match our desired arg format which has been implemented in the token_classification module, cf:  https://github.com/cleanlab/cleanlab/blob/master/cleanlab/token_classification/summary.py#L45

This should be:
```
pred_probs:
  Each row ...
```

**Comment by jwmueller** [comment]
Closing in favor of: https://github.com/cleanlab/cleanlab/pull/475

**Pull request closed without merging**